### PR TITLE
Performance Comparision of Trained models on eeg.csv

### PR DESCRIPTION
**Pull request by Addi-11**
* Various classification models have been trained on the dataset.
* Accuracy of each model measured.
* The results can be seen in [demo.ipynb]()

Further various evaluation metrics to be added and plotted for various data splits for `issue 4`.

### CODE REVIEW

**Comment by Addi-11** [comment]
* As stated in issue 3 the data was split into  various portions and evaluation metrics like -
  * Precision
  * Recall
  * Accuracy 
  * F-score
* A graph plotted to visualise the co-relation btw split and accuracy

**Comment by Addi-11** [comment]
ThankYou for your review I will work on the required changes.

**Comment by Addi-11** [comment]
I have changed and corrected my code as stated above.
I have even added graphs to help visualise and comapre the performance of all the stated models.

**Comment by Addi-11** [comment]
I will make the required changes.

**Comment by dzeber** [comment]
You still need to address the following:

- > now that you've selected the K-NN model using your evaluation scores, you should train that model on the full training set (ie. train+validation) and compute the evaluation scores for it against the test set you held out. That is your final estimate of its performance.

- > Please make sure your code runs in the environment set up in the repo. Follow steps 1-3 [here](https://github.com/mozilla/PRESC#getting-started) and run your demo notebook. Then fix any import failures.

Refresh your conda environment (delete it and recreate it) as described in the README and make sure your notebook runs without error.

**Comment by Addi-11** [comment]
I am sorry for the delay. I will work on it right away.
